### PR TITLE
Fix crash when removing a post-process stage.

### LINKED ISSUE #7210: when Removing a PostProcessStage from PostProcessStageCollection,it reports an error.
image

### PR DESCRIPTION
**Pull request by bagnell**
Fixes #7210.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @bagnell!
* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Source/Scene/PostProcessStageCollection.js [hunk 1]
```diff
@@ -294,6 +294,9 @@ define([
                 var stages = arraySlice(this._stages);
                 while (stages.length > 0) {
                     var stage = stages.pop();
+                    if (!defined(stage)) {
```
Comment (lilleyse) [inline Source/Scene/PostProcessStageCollection.js]: Is it expected that a stage in this list would be undefined? Just want to make sure this isn't a deeper issue.

Comment (bagnell) [reply Source/Scene/PostProcessStageCollection.js]: Yes. See this line in `remove`: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/PostProcessStageCollection.js#L402

They are set to `undefined` and removed on the next update.

Comment (lilleyse) [reply Source/Scene/PostProcessStageCollection.js]: Ah, ok.

**Comment by lilleyse** [comment]
This fixes the issue. Thanks @bagnell.

**Pull request merged**